__tests__/UI-tests/home/profileScreen.test.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import ProfileScreen from "@/app/screens/home/profile_screen";
+import { fireEvent, waitFor } from "@testing-library/react-native";
+import { launchImageLibraryAsync } from "expo-image-picker";
+import FirestoreCtrl, { DBUser } from "@/firebase/FirestoreCtrl";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import { resetEmail, resetPassword, logOut } from "@/types/Auth";
+
+const Stack = createNativeStackNavigator();
+
+// Mock the firestoreCtrl
+const mockFirestoreCtrl = new FirestoreCtrl();
+
+// Mock the guest user
+const mockGuestUser: DBUser = {
+  name: "Guest",
+  uid: "123",
+  email: "guest@example.com",
+  image_id: "test-image-id",
+  createdAt: new Date(),
+};
+
+// Mock the user
+const mockUser: DBUser = {
+  name: "Test User",
+  uid: "321",
+  email: "test@example.com",
+  image_id: "test-image-id",
+  createdAt: new Date(),
+};
+
+// Mock setUser
+const setUser = jest.fn();
+
```

Comment:
Great tests overall, maybe you could use a reusable setup function instead of rendering the screens in each test : 
```
const renderProfileScreen = (user: DBUser) =>
  render(<ProfileScreenTest user={user} />);
```